crates/valence_new/src/inventory.rs

```diff
@@ -159,3 +162,108 @@ pub(crate) fn update_player_inventories(mut query: Query<(&mut Inventory, &mut C
         }
     }
 }
+
+/// Used to indicate that the client with this component is currently viewing
+/// an inventory.
+#[derive(Debug, Clone, Component)]
+pub struct OpenInventory {
+    /// The Entity that has the Inventory that the client is currently
+    /// viewing.
```

Comment:
How about phrasing it like this:
```suggestion
    /// The Entity with the `Inventory` component that the client is currently
    /// viewing.
```